datadotworld/client/swagger-dwapi-def.json

```diff
@@ -871,17 +856,19 @@
           "type": "string"
         },
         "visibility": {
-          "description": "Dataset visibility. `OPEN` if the dataset can be seen by any member of data.world. `PRIVATE` if the dataset can be seen by its owner and authorized collaborators. `DISCOVERABLE` if the dataset can be seen by any member of data.world, but only files marked `sample` or `preview` are visible",
+          "description": "Dataset visibility. `OPEN` if the dataset can be seen by any member of data.world. `PRIVATE` if the dataset can be seen by its owner and authorized collaborators.",
```

Comment:
Wait, why was `DISCOVERABLE` removed?

Reply:
`DISCOVERABLE` does not appear in the public API swagger spec: https://github.com/datadotworld/platform/blob/27e8adcf742a3bf27401cb3f0ae654208905496a/dw-api/public-api/src/main/resources/META-INF/assets/swagger/swagger.json#L11015

Reply:
yes, but why was it removed in _this_ PR? That swagger spec is not directly published, and the published swagger spec may still include it.

Reply:
Replaced